src/Support/SupportService.php

```diff
@@ -0,0 +1,187 @@
+<?php
+/**
+ * Service class for support.
+ *
+ * @package AmpProject\AmpWP
+ */
+
+namespace AmpProject\AmpWP\Support;
+
+use AmpProject\AmpWP\Infrastructure\Service;
+use AmpProject\AmpWP\Infrastructure\CliCommand;
+use AmpProject\AmpWP\Validation\ScannableURLProvider;
+use AmpProject\AmpWP\Validation\URLScanningContext;
+
+/**
+ * Class SupportService
+ * Service class for support.
+ */
+class SupportService implements Service, CliCommand {
```

Comment:
This name doesn't seem ideal. Please make it more specific, like `SupportCliCommand`.